Assets/VRTK/Scripts/Controls/2D/PanelMenu/PanelMenuController.cs

```diff
@@ -106,7 +106,7 @@ protected virtual void Update()
             {
                 if (rotateTowards == null)
                 {
-                    rotateTowards = GameObject.Find("Camera (eye)");
+                    rotateTowards = VRTK_SDKManager.instance.GetHeadsetSDK().GetHeadsetCamera().gameObject;
```

Comment:
Lol i knew i'd speak too soon.. Looking at this, you should be using `VRTK_DeviceFinder.HeadsetTransform().gameObject`

Nothing should really ever go through the SDK Manager if it can help it